src/components/ChatApp/Settings/Settings.react.js

```diff
@@ -213,7 +213,7 @@ class Settings extends Component {
         console.log(errorThrown);
       },
       complete: function(jqXHR, textStatus) {
-        location.reload();
+        window.location.reload();
```

Comment:
location prop is passed by the router, not sure why you have replaced them with window.location